### PR TITLE
Use new omniauth gem

### PR DESCRIPTION
**Pull request by kristindiannefoss**
-use new gem that is still local to my file system

-remove devise and doorkeeper

-remove extra files caused by "rails scaffold" (project is not built to learn more about rails, scaffolded to get something up fast, may pull from the json info backboone to make a node app or use react.  Still nailing down plans.

### CODE REVIEW

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -1,31 +1,24 @@
...
 
   def create
-    # binding.pry
-    token = CLIENT.auth_code.get_token(params['code'], :redirect_uri => teamsnap_callback_url)
-    # binding.pry
-
-
-    @user = User.find_by(email: params[:email])
-    if @user && @user.authenticate(params[:password])
+    binding.pry
```
Comment (notmarkmiranda) [inline app/controllers/sessions_controller.rb]: prrrrrryyyyyyyy!!!!!!

Comment (kristindiannefoss) [inline app/controllers/sessions_controller.rb]: Oh.  Yeah, that.   Tanks!

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -1,23 +1,39 @@
...
-      super
-    else
-      stored_location_for(resource) || request.referer || root_path
-    end
-  end
+  helper_method :signed_in?
+  helper_method :current_user
 
+  def signed_in?
+    !!current_user
```
Comment (jeneve) [inline app/controllers/application_controller.rb:10]: This is cool- I learned something new!

**Comment by kristindiannefoss** [comment]
Not not current user, ha.  I am not not looking a ton of resources and seeing things other people have tried. This was from an exercise Josh put me on.

**Comment by kristindiannefoss** [comment]
Nate solved my issue.  I was using the code and not the token, merging to master

File: app/views/pages/index.html.erb [hunk 3]
```diff
@@ -2,6 +2,5 @@
   You are logged in as <%= current_user.email %><br>
   <%= link_to 'Log Out', logout_path, method: :delete %>
 <% else %>
-  <%= link_to 'Log In', new_session_path %><br>
-  <%= link_to 'Register', new_user_path %>
+  <%= link_to 'Log In', sign_in_path %><br>
 <% end %>
```
Comment (GKhalsa) [inline app/views/pages/index.html.erb:6]: You could take this logic out pretty easily. You could move this to a helper filer and in it's place leave <%=logged_in?%> and  <%= login_or_logout_button%>, which would call upon the appropriate methods.

File: app/controllers/application_controller.rb (hunk 2 above)
Comment (GKhalsa) [inline app/controllers/application_controller.rb]: +1

File: app/views/pages/index.html.erb (hunk 3 above)
Comment (kristindiannefoss) [inline app/views/pages/index.html.erb:6]: That's smart, thanks Sunny.  I haven't even touched the views, this is all generated by "rails scaffold", it's pretty ridiculous what it gives you.  I'm trying to focus more on the api, the gem, and try to use a rails react gem.  We'll see how that goes.  Keep em comin! Thank you!

**Pull request merged**